scene/gui/scroll_container.cpp

```diff
@@ -366,6 +367,12 @@ void ScrollContainer::_notification(int p_what) {
 
 		case NOTIFICATION_DRAW: {
 			draw_style_box(theme_cache.panel_style, Rect2(Vector2(), get_size()));
+			if (_draw_focus_border && (has_focus() || child_has_focus())) {
```

Comment:
Sorry, I didn't realize that it is editor only, in that case you will have to wrap your code inside `#ifdef TOOLS_ENABLED` and `#endif` like this.

![image](https://github.com/user-attachments/assets/943af3f9-6c35-4eb2-a175-549538fe5bbc)